runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowWorkUnitClient.java

```diff
@@ -269,4 +277,42 @@ public WorkItemServiceState reportWorkItemStatus(WorkItemStatus workItemStatus)
     logger.debug("ReportWorkItemStatus result: {}", state);
     return state;
   }
+
+  /** Reports the autoscaling signals to dataflow */
+  @Override
+  public WorkerMessageResponse reportWorkerMessage(StreamingScalingReport report)
```

Comment:
if this stays as is, the method should be renamed.  But I think it would be better if we can combine with other data sent in the same rpc.

Reply:
I think this is the first WorkerMessage sent in Legacy Worker since its a new path. Is there somewhere else where a similar RPC is sent?

Reply:
Renamed to reportStreamingMetricsWorkerMessage